### PR TITLE
[#15043][Docs] Updated the copyright year from 2020 to 2023

### PR DESCRIPTION
**Pull request by cchung100m**
Hi @yzh119 @Hzfengsy 

This is a minor PR and it aims to update the copyright year from 2020 to 2023. I would appreciate that if you can help to review/manage it, thanks.

#15043

### CODE REVIEW

File: licenses/LICENSE.cutlass.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2017 - 2020, NVIDIA CORPORATION.  All rights reserved.
+Copyright (c) 2017 - 2023, NVIDIA CORPORATION.  All rights reserved.
```
Comment (yzh119) [inline licenses/LICENSE.cutlass.txt:1]: Is it necessary to update the content of the license?
There are also many other license files that include years, and I don't we need to update them all.

**Review by yzh119 — COMMENTED** [review]
Seems the footer of https://tvm.apache.org/ is not changed in this PR.

**Comment by cchung100m** [comment]
> Seems the footer of https://tvm.apache.org/ is not changed in this PR.

Hi @yzh119 

I thought the footer is here. Please correct me if I am wrong.

https://github.com/apache/tvm/pull/15071/files#diff-85933aa74a2d66c3e4dcdf7a9ad8397f5a7971080d34ef1108296a7c6b69e7e3R595

File: licenses/LICENSE.cutlass.txt (hunk 1 above)
Comment (cchung100m) [reply licenses/LICENSE.cutlass.txt]: Thank you @yzh119. Good catch!  I updated the part you mentioned.

**Comment by yzh119** [comment]
Hi @cchung100m , the text in the footer is "2020 Apache Software Foundation" and "Copyright © 2020 The Apache Software Foundation", and I think this PR fixes year 2022 to 2023 which do not involve 2020.

**Comment by cchung100m** [comment]
> Hi @cchung100m , the text in the footer is "2020 Apache Software Foundation" and "Copyright © 2020 The Apache Software Foundation", and I think this PR fixes year 2022 to 2023 which do not involve 2020.

Hi @yzh119 

Thanks for the prompt reply. I search the whole project and did not see the `2020`. Is it possible that the website page with copyright year 2020 is cached somewhere?

**Comment by yzh119** [comment]
Seems the website is hosted in another project, and you can create another PR to https://github.com/apache/tvm-site and change these two lines:
https://github.com/apache/tvm-site/blob/c0c11309c181a0e86a896d2d9b0508c92480980a/_layouts/default.html#L41
https://github.com/apache/tvm-site/blob/c0c11309c181a0e86a896d2d9b0508c92480980a/_layouts/default.html#L49

**Review by yzh119 — APPROVED** [review]
LGTM, thanks for doing this.

**Comment by cchung100m** [comment]
> Seems the website is hosted in another project, and you can create another PR to https://github.com/apache/tvm-site and change these two lines: https://github.com/apache/tvm-site/blob/c0c11309c181a0e86a896d2d9b0508c92480980a/_layouts/default.html#L41 https://github.com/apache/tvm-site/blob/c0c11309c181a0e86a896d2d9b0508c92480980a/_layouts/default.html#L49

Hi @yzh119 

Thanks for your friendly reminder and I will submit another PR for it.

**Comment by cchung100m** [comment]
Hi @yzh119 

The failed built jobs seem that irrelevant to this PR. If convenient, may I ask how to solve it?

**Comment by yzh119** [comment]
It appears that the VTA CI is not functioning properly. @cchung100m, would you kindly refrain from making changes in the "3rdparty/vta-hw" directory? Ideally, we should avoid altering the contents of any submodules.

**Comment by cchung100m** [comment]
Hi @yzh119 

Thanks for the kindly reminder and I updated the part you mentioned.

**Pull request merged**